src/index.js

```diff
@@ -0,0 +1,70 @@
+import './style.css';
+
+const taskSection = document.querySelector('.tasks');
+const userTask = document.querySelector('.user-task');
+const tasksArray = JSON.parse(localStorage.getItem('task')) || [];
+let MOOD = 'CREATE';
+let tmp;
+class Task {
+  constructor(userTask) {
+    this.id = tasksArray.length;
```

Comment:
- [ ] The id of the array should start from `1`, if a 5th task is added to the list, the index of that task should equal to `5`. In your case the array starts from `0` and 5th element would have id of `4`. Kindly make the id start from 1. You an simply do it by `this.id = tasksArray.length + 1` 😉